src/types/reactivity.ts

```diff
@@ -1,7 +1,9 @@
 import { Ref } from 'vue'
 import { AnyArray, AnyRecord } from '@/types/any'
 
+export type Getter<T> = () => T
```

Comment:
Oh no wonder I couldn't import this type from Vue lol

Reply:
I was very surprised they didn't export this.